style.css

```diff
@@ -227,3 +406,141 @@ header {
 #times {
   display: none;
 }
+
+/* Popup window */
+.popup-container {
+  background-color: #03030349;
+  width: 100%;
+  height: 100%;
+  display: none;
+  top: 0%;
+  position: fixed;
+  z-index: 3;
+  justify-content: center;
+  align-items: center;
+}
+
+.popup-window {
+  background-color: #fff;
+  display: block;
+  width: 90%;
+  height: 96%;
+  flex-direction: column;
+  margin: 20px 2px;
+}
+
+.mx-20 {
+  margin: 0 10px !important;
+}
+
+.popup-top-content {
+  padding: 0 10px;
+  justify-content: space-between;
+}
+
+.popup-window .box-container {
+  margin-top: -5% !important;
+}
+
```

Comment:
- [ ] It would really be nice if you can avoid the temptation of overusing `!important` rule in your CSS, this is also a good [HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) practice